apps/docs/src/guide/introduction/getting-started.md

```diff
@@ -93,42 +93,46 @@ For a quick test or just playing around, you can load it in your Web Apps straig
 
 At a high level, you can use the Fuel TypeScript SDK to build applications that can run computations on the Fuel Virtual Machine through interactions with smart contracts written in Sway.
 
-For this interaction to work, the SDK must be able to communicate with a `fuel-core` node; you have two options at your disposal:
+For this interaction to work, the SDK must be able to communicate with a [`fuel-core`](https://github.com/FuelLabs/fuel-core) node; you have two options at your disposal:
 
-1. Use the [Testnet](#connecting-to-the-testnet). (For application building)
+1. Connecting to the [Testnet](#connecting-to-the-testnet). (For application building)
 2. Running a [local node](https://docs.fuel.network/guides/running-a-node/). (For smart contract testing)
 
 
 ### Connecting to the Testnet
 
-We can interact with the `Testnet` node by using the following example.
+The **Testnet** is a public network that allows you to interact with a Fuel Virtual Machine (managed by Fuel) and is used predominantly for application building.
 
-<<< @/../../../packages/fuel-gauge/src/doc-examples.test.ts#provider-testnet{ts:line-numbers}
+> [!NOTE] Latest Testnet
+> Beta 5
+>
+> `https://beta-5.fuel.network/graphql`
 
-In the code example, we connected a new provider to the Testnet node and created a new wallet from a private key.
+We have some useful resources for the Testnet:
 
-> **Note:** New wallets on the Testnet will not have any assets! They can be obtained by providing the wallet address to the faucet at
->
-> [faucet-beta-5.fuel.network](https://faucet-beta-5.fuel.network/)
->
-> Once the assets have been transferred to the wallet, you can reuse it in other tests by providing the private key!
->
-> In addition to the faucet, there is a block explorer for the Testnet at
->
-> [block-explorer](https://fuellabs.github.io/block-explorer-v2)
+- [**Faucet**](https://faucet-beta-5.fuel.network/) - for funding wallets that have been created.
+- [**Explorer**](https://next-app.fuel.network/) - for viewing transactions and blocks.
+- [**Playground**](https://beta-5.fuel.network/playground) - for testing GraphQL queries and mutations.
+
+---
+
+With the example below, we connect a [Provider](../providers/index.md) to the Latest Testnet node and create a new wallet from a private key.
```

Comment:
```suggestion
In the example below, we connect a [Provider](../providers/index.md) to the latest testnet and create a new wallet from a private key.
```

- "In the example below" is IMO better than "With the example below". "With" feels too prescriptive. 
- "Latest Testnet node" -> "latest testnet node"
    - Latest Testnet shouldn't be capitalized
    - The testnet is a collection of nodes so it's best to just not mention "node" at all